lib/modules/comfy_opengraph.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
```

Comment:
Have a look at `app/controllers/comfy/cms/content_controller.rb` to see where this class gets utilised.
The controller is the overridden controller from Comfy - I've made it obvious where I've made modifications with the comment blocks. Where a page is loaded, it will always have `@cms_page` set, if this helps clarify things?